test/CalendarHeatmap.test.jsx

```diff
@@ -177,6 +177,28 @@ describe('CalendarHeatmap props', () => {
     expect(vertical.find('text.react-calendar-heatmap-small-text')).toHaveLength(3);
   });
 
+  it('showAllWeekdayLabels true', () => {
+    const calendar = shallow(<CalendarHeatmap
+      startDate={dateNDaysAgo(7)}
+      values={[]}
+      showWeekdayLabels={true}
```

Comment:
do you think folks should have to pass both `showWeekdayLabels` and `showAllWeekdayLabels` or should just `showAllWeekdayLabels` be enough? I like the idea of just passing `showAllWeekdayLabels` by itself rather than 2 props, I think you have to update this line https://github.com/patientslikeme/react-calendar-heatmap/pull/87/files#diff-73642d587163f2eb4d72f0fa6ebfba02L293 to make a single prop enough